cypress/integration/aaaaaaaaaaa/manufacturing_with_issuing_from_multiple_CUs.js

```diff
@@ -156,19 +188,146 @@ describe('Create Manufacturing Order', function() {
+    selectHuAsSource(productComponentName3, huValue3);
+  });
+
+  it('Receive the finished product', function() {
+    cy.selectRowByColumnAndValue({ column: productColumn, value: finishedProductName }, true);
+    cy.executeQuickAction('WEBUI_PP_Order_Receipt', true);
+    cy.getStringFieldValue('M_HU_PI_Item_Product_ID').should('contain', expectNoPackingItem);
+    cy.writeIntoStringField('QtyCU', manufacturingQtyEntered, true);
+    cy.pressStartButton();
+    cy.selectRowByColumnAndValue(createColumnAndValue(finishedProductName, undefined, 'CU', null, manufacturingQtyEntered, packingStatusPlanning), true);
+  });
+
+  it('Run action "Issue CUs from source HUs" when only component 1 is selected', function() {
+    cy.selectRowByColumnAndValue(createColumnAndValue(productComponentName1, null, 'CO'), true);
+    cy.executeQuickAction('WEBUI_PP_Order_M_Source_HU_IssueCUQty', true);
+    cy.getStringFieldValue('QtyCU').should('equals', productComponentQty1.toString(10));
+    cy.pressStartButton();
+    cy.selectRowByColumnAndValue(createColumnAndValue(productComponentName1, undefined, 'CU', null, productComponentQty1, packingStatusIssued), true);
+    cy.selectRowByColumnAndValue(createColumnAndValue(productComponentName1, undefined, 'CU', null, huQty - productComponentQty1, packingStatusActive), true);
+  });
+
+  it('Run action "Issue CUs from source HUs" when both component 2 and 3 are selected', function() {
+    // multiple selection by pressing shift and clicking
+    cy.selectRowByColumnAndValue(createColumnAndValue(productComponentName2, null, 'CO'), true)
+      .get('body')
+      .type('{shift}', { release: false })
+      .selectRowByColumnAndValue(createColumnAndValue(productComponentName3, null, 'CO'), true)
+      .get('body')
+      .type('{shift}');
 
+    cy.executeQuickAction('WEBUI_PP_Order_M_Source_HU_IssueCUQty', true);
+    // i have the feeling this 500ms sleep may in certain cases not be enough, however currently i have no idea how "not to need" it.
+    // note in case you want to wait for a `RewriteURL.DocumentLayout`: that's not gonna work, i have already tried that.
+    // there are some cases where a new layout is not needed, even though something new appears onscreen. ¯\_(ツ)_/¯
+    cy.pressStartButton(500);
+
+    cy.selectRowByColumnAndValue(createColumnAndValue(productComponentName2, undefined, 'CU', null, productComponentQty2, packingStatusIssued), true);
+    cy.selectRowByColumnAndValue(createColumnAndValue(productComponentName2, undefined, 'CU', null, huQty - productComponentQty2, packingStatusActive), true);
+
+    cy.selectRowByColumnAndValue(createColumnAndValue(productComponentName3, undefined, 'CU', null, productComponentQty3, packingStatusIssued), true);
+    cy.selectRowByColumnAndValue(createColumnAndValue(productComponentName3, undefined, 'CU', null, huQty - productComponentQty3, packingStatusActive), true);
   });
 
+  it('Process the finished product', function() {
```

Comment:
What does a finished product means?

Reply:
It refers to "finished goods", so maybe "finishedGood" is better than "finished product"